lib/galaxy/tool_util/xsd/galaxy.xsd

```diff
@@ -4442,6 +4448,7 @@ used to generate dynamic options.
       <xs:element name="filter" type="Filter" minOccurs="0" maxOccurs="unbounded"/>
       <xs:element name="column" type="Column" minOccurs="0" maxOccurs="unbounded"/>
       <xs:element name="validator" type="Validator" minOccurs="0" maxOccurs="1"/>
+      <xs:element name="postprocess_expression" type="Expression" minOccurs="0" maxOccurs="1"/>
```

Comment:
Maybe link the test tool from here?

Maybe a linter checking that `postprocess_expression` only appears for `from_url` could be nice. But this would only make sense with the big linter PR merged.